src/obstacle_params_parser.cpp

```diff
@@ -0,0 +1,157 @@
+#pragma once
+
+#include "panda_torque_mpc/obstacle_params_parser.h"
+
+#include <memory>
+#include <vector>
+
+#include <pinocchio/fwd.hpp>
```

Comment:
Can you bring back the imports for the boost, pinocchio geometry and also add iostream?